SYCL/Basic/buffer/subbuffer.cpp

```diff
@@ -3,6 +3,9 @@
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
+
+// UNSUPPORTED: (opencl && gpu)
```

Comment:
All changes in this file doesn't look related to SYCL2020 errc conformance, I couldn't find any errc checks in the file.
It is possible to move them to a separate PR if they are unrelated? Does it make sense to create a separate test for your modified test case (which shows a problem on opencl gpu) to avoid disabling whole test for opencl gpu?

Reply:
This test was using an out-of-bounds range argument to an accessor. And, oddly, that was masking an error that is occurring over on OCL:GPU. I reduced that down and reported it, there is a ticket against it now. But in the interim, this test won't pass on OCL:GPU, not due to any bug in the error system, but due to a real bug that OCL:GPU has handling offsets.  Strangely, that is the only device with the problem. L0, OCL:CPU, OCL:ACC and Host are all fine.